app/views/admin/users/_form.html.haml

```diff
@@ -0,0 +1,23 @@
+= form_for @user, url: admin_users_path, method: 'post' do |form|
```

Comment:
`post` means it will go to `create` action, to call `update` action you should set method to `put` or `patch`

Reply:
btw AFAIK form_for will automagically set proper method based on id presence.

Reply:
are you sure it's the case when specifying custom url?